src/main/java/Ui.java

```diff
@@ -0,0 +1,89 @@
+import java.io.IOException;
+
+public class Ui {
+    String userInput;
+    Parser parser;
+
+    public static final String LIST = "list";
+    public static final String ADD = "add";
+    public static final String DELETE = "delete";
+    public static final String DONE = "done";
+    public static final String FIND = "find";
+
+
+    /**
+     * Constructor for Ui.
+     */
+    public Ui() {
+        //deals with interactions with the user
+        //this.userInput = userInput;
+
+        parser = new Parser();
+    }
+
+
+    /**
+     * Use Parser to read user's command to call correct method, and prints out any error encountered.
+     *
+     * @param command User's input.
+     * @param tasks TaskList class created in Duke.
+     * @throws IOException Throws away the exception.
+     */
+    public String readCommand(String command, TaskList tasks) throws IOException {
+        String commandArr = parser.makeSenseOfUserCommand(command);
```

Comment:
Try to keep short method names if possible, lest you'll have to keep typing long names.
Something like parseCommand is sufficient.